contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -850,18 +849,18 @@ TEST(Config, TestCorsDisabled) {
   MockApiManagerEnvironmentWithLog env;
 
   static const char config_text[] = R"(
- name: "Service.Name"
- http {
-   rules {
-     selector: "CORS"
-     get: "/shelves"
-   }
-   rules {
-     selector: "CORS.1"
-     get: "/shelves/{shelf}"
-   }
- }
-)";
+    name: "Service.Name"
+    http {
+      rules {
```

Comment:
The style is recommended by Google dev guide
https://screenshot.googleplex.com/eLDuLGnpUNd